unyt/__init__.py

```diff
@@ -96,7 +96,9 @@
 from unyt.unit_systems import UnitSystem  # NOQA: F401
 from unyt.testing import assert_allclose_units  # NOQA: F401
 from unyt.dimensions import accepts, returns  # NOQA: F401
-from unyt.mpl_interface import MplUnitsCM  # NOQA: F401
+from unyt.mpl_interface import matplotlib_support  # NOQA: F401
+
+matplotlib_support = matplotlib_support()
```

Comment:
Won’t this turn on matplotlib support if someone imports unyt?

Reply:
no.
```python
>>> import unyt
>>> import matplotlib.units as mplunits
>>> mplunits.registry
{}
>>> with matplotlib_support:
        print(mplunits.registry)
{unyt_array...}
>>> mplunits.registry
{}
```